platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/AttributeFilter.java

```diff
@@ -146,42 +145,46 @@ private ProjectTypes getProjectTypes(FolderEntry module, ProjectConfig moduleCon
                                                                                                 ServerException {
         Project project = new Project(module, projectManager);
 
-        ProjectTypes types = new ProjectTypes(project, moduleConfig.getType(), moduleConfig.getMixins(), projectManager);
-        types.addTransient();
+        ProjectTypes types = null;
+        try {
+            types = new ProjectTypes(project, moduleConfig.getType(), moduleConfig.getMixins(), projectManager);
+            types.addTransient();
+        } catch (NotFoundException e) {
+            e.printStackTrace();
```

Comment:
It is a bad practice to print stack trace instead of logging or processing exception